<?xml version="1.0"?>
<launch>

  <arg name="address"		default="10.66.171.52"/>
  <arg name="namespace"		default="gripper"/>
  <arg name="gripper_prefix"	default=""/>
  <arg name="config"		default="cmodel_action_controller"/>

  <group ns="$(arg namespace)">
    <node name="cmodel_urcap_driver"
	  pkg="aist_robotiq" type="cmodel_urcap_driver.py"
	  output="screen" args="$(arg address)"/>

    <param name="gripper_prefix" value="$(arg gripper_prefix)"/>
    <rosparam file="$(dirname)/../config/$(arg config).yaml"/>
    <node name="cmodel_action_controller"
	  pkg="aist_robotiq" type="cmodel_action_controller.py"
	  output="screen"/>
  </group>

</launch>
